lib/pdk/module/metadata.rb

```diff
@@ -30,32 +30,32 @@ class Metadata
         'Debian based Linux' => [
           {
             'operatingsystem'        => 'Debian',
-            'operatingsystemrelease' => ['8'],
+            'operatingsystemrelease' => ['9'],
           },
           {
             'operatingsystem'        => 'Ubuntu',
-            'operatingsystemrelease' => ['16.04'],
+            'operatingsystemrelease' => ['18.04'],
           },
         ],
         'Fedora' => {
           'operatingsystem'        => 'Fedora',
-          'operatingsystemrelease' => ['25'],
+          'operatingsystemrelease' => ['29'],
         },
         'OSX' => {
           'operatingsystem'        => 'Darwin',
           'operatingsystemrelease' => ['16'],
         },
         'SLES' => {
           'operatingsystem'        => 'SLES',
-          'operatingsystemrelease' => ['12'],
+          'operatingsystemrelease' => ['15'],
         },
         'Solaris' => {
           'operatingsystem'        => 'Solaris',
           'operatingsystemrelease' => ['11'],
         },
         'Windows' => {
           'operatingsystem'        => 'windows',
-          'operatingsystemrelease' => ['2008 R2', '2012 R2', '10'],
+          'operatingsystemrelease' => %w[2019 10],
```

Comment:
This one is a bit tricky. The most common Server OS right now is either 2016 or 2012R2, but 2019 is the latest.